services/comparison.py

```diff
@@ -774,12 +777,8 @@ def _fetch_comparison_and_reverse_comparison(self):
             self.base_commit.commitid, self.head_commit.commitid
         )
 
-        reverse_comparison_coro = adapter.get_compare(
```

Comment:
This other routine was doing an extra Github API call every time we needed a fresh version of this property (assuming the caching worked to begin with)

Reply:
Hmmm how do we know we don't need this still?

Reply:
The return value from this thing was only being used in "has_unmerged_base_commits" which didn't seem to be used anywhere